src/sleep/sleep.c

```diff
@@ -350,10 +350,18 @@ static int custom_timer_suspend(const SleepConfig *sleep_config) {
 }
 
 static int execute_s2h(const SleepConfig *sleep_config) {
+        _cleanup_(freeze_thaw_user_slice) const char *method = "thaw";
```

Comment:
The method call is called FreezeUnit() and ThawUnit(), not thaw() and freeze()

Reply:
I think we should give this a better name. i.e. `auto_method_thaw` or so?

Reply:
Ahh. So sorry about that. Confused it with the verb in systemctl and then totally missed to set it. Fixing it now.